spec/importc.dd

```diff
@@ -600,6 +600,23 @@ $(H2 $(LNAME2 gnu-clang-extensions, Gnu and Clang Extensions))
     $(P `gcc` and `clang` are presumed to have the same behavior w.r.t. extensions,
     so `gcc` as used here refers to both.)
 
+
+    $(H3 $(LNAME2 __attribute__, `__attribute__` Extensions))
+
+    $(P The following $(LINK2 https://gcc.gnu.org/onlinedocs/gcc/Function-Attributes.html, `__attribute__` extensions):)
+
+    $(OL
+    ($LI `__attribute__((aligned(N)))`)
+    ($LI `__attribute__((always_inline))`)
+    ($LI `__attribute__((deprecated))`)
+    ($LI `__attribute__((dllexport))`)
+    ($LI `__attribute__((dllimport))`)
+    ($LI `__attribute__((naked))`)
+    ($LI `__attribute__((noinline))`)
+    ($LI $(RELATIVE_LINK2 noreturn, `__attribute__((noreturn))`))
+    $(LI others are ignored)
+    )
+
```

Comment:
The formatting looks off:
https://dtest.dlang.io/artifact/website-fce365a6f13559097ea26962291e0bf5ce8a8f3f-e8f708aabc1b0dac72d4102dff4917f8/web/spec/importc.html#__attribute__

Reply:
Looks fine to me :-)